lms/envs/aws.py

```diff
@@ -400,6 +399,13 @@
 
 AWS_STORAGE_BUCKET_NAME = AUTH_TOKENS.get('AWS_STORAGE_BUCKET_NAME', 'edxuploads')
 
+if AUTH_TOKENS.get('DEFAULT_FILE_STORAGE'):
```

Comment:
Yes, globally there is a fair amount of duplication in the env files on the platform repo, and then some more in the configuration repository. Would be great to refactor it all at some point.